R/vignette.R

```diff
@@ -13,40 +13,60 @@
 #' @param name Base for file name to use for new vignette. Should consist only
 #'   of numbers, letters, `_` and `-`. Lower case is recommended.
 #' @param title The title of the vignette.
+#' @param type One of `"quarto"` or `"rmarkdown"`
```

Comment:
agreed that `type` may not be ideal. If we were to go with vuilder, the arguments should be updated to "knitr" instead of "rmarkdown"?


Maybe `ext = c("qmd", "rmd")` would work too. 

Maybe there should be an option

`options(usethis.vignette = "quarto")` to avoid specifying it everytime?

I can imagine myself deciding to stick with a single type, and not have to specify it everytime.

Reply:
I like `ext`, and an option would be great!